lib/ui.rb

```diff
@@ -42,9 +38,24 @@ def selected_ship
   end
 
   def coordinates_and_orientation
-    @stdout.puts "Choose 2 coordinates X,Y and an orientation 'horizontal' or 'vertical' (ex. 2,b,vertical)"
+    @stdout.puts "Choose 2 coordinates X,Y and an orientation h for 'horizontal' or v for 'vertical' (ex. 2,b,h)"
     input = @stdin.gets.chomp.split(",")
```

Comment:
If you wanted to use symbols for :horizontal and :vertical, I think the Ui would be the place to convert the user input "h" and "v" into the symbols. And then the rest of your application can just deal with the symbols and does not need to care about the user input. 
For example, if you decide to change what the user inputs and go for "1" and "2" instead, you will have to change it all over your application. But if you convert it into symbols in this class, then you only need to change that one method.